app/src/main/java/org/simple/clinic/forgotpin/confirmpin/ForgotPinConfirmPinModel.kt

```diff
@@ -4,4 +4,9 @@ import android.os.Parcelable
 import kotlinx.android.parcel.Parcelize
 
 @Parcelize
-class ForgotPinConfirmPinModel : Parcelable
+class ForgotPinConfirmPinModel : Parcelable {
+
+  companion object {
```

Comment:
Should the companion object be added in `Add no-op Mobius classes` commit?

Reply:
Since we will only start using it once we set up the test fixture it doesn't matter. I personally preferer adding during this commit but we can also add it in no-op if we want.